<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-2O8UC2F" failures="0" tests="1" name="producttest.Testcase_10" time="6.490" errors="1" timestamp="2025-05-05T15:19:21 IST" skipped="0">
  <testcase classname="producttest.Testcase_10" name="verifysubscriptioninhome" time="6.490">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebElement.click()&quot; because the return value of &quot;pagerepository_A.HomePage.getsubscribe()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.click()" because the return value of "pagerepository_A.HomePage.getsubscribe()" is null
at producttest.Testcase_10.verifysubscriptioninhome(Testcase_10.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- verifysubscriptioninhome -->
  <system-out>
    <![CDATA[Closebrowser Successfully
checkout
Closebrowser Successfully
checkout
Closebrowser Successfully
Closebrowser Successfully
Closebrowser Successfully
verify products
Closebrowser Successfully
remove product
Closebrowser Successfully
viewcart
Closebrowser Successfully
login_with_invalid_credentials
Closebrowser Successfully
Closebrowser Successfully
user is logged in with valid credentilas
Closebrowser Successfully
Closebrowser Successfully
Closebrowser Successfully
user  is logged out
Closebrowser Successfully
Closebrowser Successfully
Test Case 21: Add review on product
Closebrowser Successfully
viewcart
Closebrowser Successfully
user is registered
Closebrowser Successfully
testcase 23
Closebrowser Successfully
contact form is created
Closebrowser Successfully
Closebrowser Successfully
contact form is created
Closebrowser Successfully
product is searched
Closebrowser Successfully
contact form is created
Closebrowser Successfully]]>
  </system-out>
</testsuite> <!-- producttest.Testcase_10 -->
